instruqt-tracks/terraform-cloud-aws-v2/09-vcs-collaboration/check-workstation

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -l
+set -e
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
```

Comment:
Any possibilities for checking results here, even if only the very end result somehow?